gemma-web/src/main/webapp/WEB-INF/web.xml

```diff
@@ -75,45 +75,29 @@
         <filter-class>com.opensymphony.module.sitemesh.filter.PageFilter</filter-class>
     </filter>
 
-    <!-- Filter mappings -->
-    <filter-mapping>
-        <filter-name>exceptionHandlerFilter</filter-name>
-        <url-pattern>/rest/v2/*</url-pattern>
-    </filter-mapping>
-
+    <!-- Spring Security -->
     <filter-mapping>
-        <filter-name>encodingFilter</filter-name>
-        <url-pattern>/j_spring_security_check</url-pattern>
-    </filter-mapping>
-
-    <filter-mapping>
-        <filter-name>encodingFilter</filter-name>
-        <url-pattern>/dwr/*</url-pattern>
+        <filter-name>springSecurityFilterChain</filter-name>
+        <url-pattern>/*</url-pattern>
     </filter-mapping>
 
+    <!-- Filter mappings -->
     <filter-mapping>
-        <filter-name>encodingFilter</filter-name>
-        <url-pattern>*.html</url-pattern>
+        <filter-name>exceptionHandlerFilter</filter-name>
+        <servlet-name>gemma-rest</servlet-name>
     </filter-mapping>
 
+    <!-- enforce UTF-8 -->
     <filter-mapping>
         <filter-name>encodingFilter</filter-name>
-        <url-pattern>*.jsp</url-pattern>
-    </filter-mapping>
-
-    <!-- Security -->
-    <filter-mapping>
-        <filter-name>springSecurityFilterChain</filter-name>
-        <url-pattern>/*</url-pattern>
+        <url-pattern>/j_spring_security_check</url-pattern>
+        <servlet-name>gemma</servlet-name>
     </filter-mapping>
 
-
     <!-- All requests are decorated with sitemesh -->
     <filter-mapping>
         <filter-name>sitemesh</filter-name>
-        <url-pattern>/*</url-pattern>

```

Comment:
@ppavlidis that was the culprit: it was intercepting the index.html from swagger-ui, but now it only applies to the gemma servlet